packages/tabs-react/documentation/Tabs.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Tabs
+react: tabs-react
+scss: tabs
+group: knapper
+---
+
+import TabsExample, { tabsExampleCode } from "./TabsExample";
+import "@fremtind/jkl-tabs/tabs.css";
+
+<Ingress>En liste med tabs kan brukes til å bytte mellom innhold på en side.</Ingress>
+
+<ComponentExample title="Tabs" component={TabsExample} codeExample={tabsExampleCode} />
+
+## Bruk
+
+-   Denne komponenten egner seg ikke som hovednavigasjon i en applikasjon.
+-   Unngå å bruk for mange nivåer med tabs, og bruk de helst ikke rett under hverandre.
```

Comment:
Liten trykkleif. Men hvor mange er "for mange"? To, tre, fem, tretten? 😄 
```suggestion
-   Unngå å bruke for mange nivåer med tabs, og bruk de helst ikke rett under hverandre.
```

Reply:
Hvis vi først pirker på Leif:

> bruk ~de~ **dem** helst ikke

Og angående hvor mye som er for mange er det kanskje ikke naturlig å sette et bestemt tall, med mindre det er null? Dette er nok igjen noe en interaksjonsdesigner kanskje bør si noe om, men kanskje man bare kunne sagt:

```suggestion
-   Unngå helst å bruke flere nivåer av tabs, og bruk dem ikke rett under hverandre.
```